docs/markdown/Introduction/media.md

```diff
@@ -4,89 +4,182 @@ slug: "media"
 excerpt: "Learn more about Pants and related topics from these talks, posts and podcasts featuring Pants contributors and users."
 hidden: false
 createdAt: "2021-04-18T17:27:18.361Z"
-updatedAt: "2022-09-08T22:02:38.457Z"
+updatedAt: "2023-01-04T22:02:38.457Z"
 ---
-## Posts & Articles
 
-### Doctrine.fr Engineering blog
+## Case Studies
 
-Nov 25, 2022  
-[**Tutorial**] "Industrialized Python code with Pylint plugins in Pants"  
-<https://medium.com/doctrine/industrialized-python-code-with-pylint-plugins-in-pants-321d9cbad07a>
+### Tweag
 
-> What if you needed to put strict code convention rules through a linter and apply them with the help of Pants?
+January 3, 2023
+"From adopting Pants, to generalizing our CI to multiple Python versions" <https://blog.pantsbuild.org/tweag-case-study/>
+
+> Describes why and how the maintainers of Bazel's Haskell rules chose to use Pants for a client's Python monorepo. Also covers some gotchas for newcomers.
 
-### Coinbase Blog
+### Oxbotica
+
+November 4, 2022
+"Introducing Pants to Oxbotica"
+
+> Self-driving vehicle company's successful journey to adopting Pants locally and in CI.
+
+
+### Coinbase
 
 September 1, 2022  
-[**Case Study**] "Building a Python ecosystem for efficient and reliable development"  
+"Part 1: Building a Python ecosystem for efficient and reliable development"  
 <https://blog.coinbase.com/building-a-python-ecosystem-for-efficient-and-reliable-development-d986c97a94a0>
 
-> Part 1 of 2. Describes how the company developed an efficient, reliable Python ecosystem using Pants, and solved the challenge of managing Python applications at a large scale at Coinbase.
+> Describes how the company developed an efficient, reliable Python ecosystem using Pants, and solved the challenge of managing Python applications at a large scale at Coinbase.
+
+
+October 27, 2022
+"Part 2: Building a Python ecosystem for efficient and reliable development" <https://www.coinbase.com/blog/part-2-building-a-python-ecosystem-for-efficient-and-reliable-development>
```

Comment:
According to [the preview](https://github.com/pantsbuild/pants/blob/24075710a6a3d0ac6faca2e77b7b795684ece037/docs/markdown/Introduction/media.md), this needs two trailing spaces to actually result in newlines.
```suggestion
October 27, 2022  
"Part 2: Building a Python ecosystem for efficient and reliable development"  
<https://www.coinbase.com/blog/part-2-building-a-python-ecosystem-for-efficient-and-reliable-development>
```

Reply:
Thanks, fixed.